### PR TITLE
Update to stable Cadence

### PR DESCRIPTION
**Pull request by SupunS**
Feature branch for stable cadence updates to integrate the new token standards, core contracts, and other features of stable cadence

Depends on https://github.com/onflow/flow-go/pull/4655

### CODE REVIEW

**Comment by SupunS** [comment]
There are some tests failing that seem to be unrelated to the cadence change. Can someone with more context have a look please?

File: cmd/util/cmd/common/transactions.go [hunk 1]
```diff
@@ -10,7 +10,7 @@ import (
 const (
 	getInfoForProposedNodesScript = `
 		import FlowIDTableStaking from 0x%s
-		pub fun main(): [FlowIDTableStaking.NodeInfo] {
+		access(all) fun main(): [FlowIDTableStaking.NodeInfo] {
```
Comment (turbolent) [inline cmd/util/cmd/common/transactions.go:13]: Maybe add line breaks after access modifiers, as they are now quite long, making the function signature hard to read

File: integration/utils/templates/create-and-setup-node.cdc [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import Crypto
 import FungibleToken from 0xFUNGIBLETOKENADDRESS
 import FlowToken from 0xFLOWTOKENADDRESS
-import FlowIDTableStaking from 0xIDENTITYTABLEADDRESS
+import FlowIDTableStaking from "FlowIDTableStaking"
```
Comment (turbolent) [inline integration/utils/templates/create-and-setup-node.cdc:4]: Can the other `0xFOO` "addresses" in the other imports also be replaced to the string literal?

**Review by turbolent — APPROVED** [review]
Nice!

File: cmd/util/cmd/common/transactions.go (hunk 1 above)
Comment (joshuahannan) [reply cmd/util/cmd/common/transactions.go:13]: idk, I feel like that makes the code looks pretty awkward

File: integration/utils/templates/create-and-setup-node.cdc (hunk 2 above)
Comment (joshuahannan) [reply integration/utils/templates/create-and-setup-node.cdc:4]: yeah, I'm working on it. I won't be able to get to it in this PR, but maybe in the next one

**Review by joshuahannan — APPROVED** [review]

**Comment by SupunS** [comment]
There are still two tests failing:
- `TestPassThrough/TestSealingAndVerificationPassThrough` in  `Integration Tests (make -C integration bft-tests)`
- `TestFullGossipSubConnectivityAmongHonestNodesWithMaliciousMajority` in `Unit Tests (network)`

Don't have much clue on why they are failing / how to fix.

**Comment by SupunS** [comment]
bors merge

File: cmd/util/cmd/common/transactions.go (hunk 1 above)
Comment (SupunS) [reply cmd/util/cmd/common/transactions.go:13]: I will update the tests where necessary in a follow-up PR.

**Pull request merged**